frontend/app/components/address.tsx

```diff
@@ -8,26 +8,33 @@ export interface AddressProps extends ComponentProps<'address'> {
   provinceState?: string;
   postalZipCode?: string;
   country: string;
+  altFormat?: boolean;
 }
 
-function formatAddress(address: string, city: string, country: string, provinceState?: string, postalZipCode?: string) {
+function formatAddress(address: string, city: string, country: string, provinceState?: string, postalZipCode?: string, altFormat?: boolean) {
   // TODO 'canada' shouldn't be hardcoded as we may deal with different values of the country field such as abbreviations
   const isNotCanadianAddress = 'canada' !== country.toLowerCase();
 
   // prettier-ignore
   const lines = [`${address}`,
-    `${city}${provinceState ? ` ${provinceState}` : ''}${postalZipCode ? `  ${postalZipCode}` : ''}`,
+    `${city}${provinceState ? ` ${provinceState}` : ''}${postalZipCode ? ` ${postalZipCode}` : ''}`,
```

Comment:
The extra space should be left as is if this default format is the Canada Post address format.



> Separate the postal code from the province symbol by 2 spaces.

See https://www.canadapost-postescanada.ca/cpc/en/support/kb/business/address-accuracy/addressing-mail-accurately

Reply:
Re-added spacing